code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -708,10 +801,10 @@
 			M.drop_item()
 	if(prob(50))
 		update_flags |= M.adjustFireLoss(10, FALSE)
-	update_flags |= M.adjustBrainLoss(pick(0.5, 0.6, 0.7, 0.8, 0.9, 1), FALSE)
+	update_flags |= M.adjustBrainLoss(20, FALSE)
```

Comment:
Overdose happens whenever the amount of chemicals inside of you is greater than `overdose_threshold`, and trigger on each tick.
This means that injecting 25+ units of lube is a guaranteed death in 5 cycles with pretty much no way at all to fix it (especially for IPCs that can neither dialysis nor have charcoal to save them; Voxes are *a bit* more fixable but still extremely rough)
It feels way too brutal.

Please add a comment if intentional.

Reply:
Minor note, but iirc degreaser acts as a purging chem for IPCs, especially against lube and ultra lube.
A brainloss of 20 per tick is still massive so I think it's a fair concern, purge or not.